.github/workflows/build_wheels.yml

```diff
@@ -20,16 +20,11 @@ jobs:
       - name: Install requirements
         run: python -m pip install cibuildwheel==2.11.4 twine==4.0.2
 
-      - name: Init pybind11 submodule
-        run: |
-          git submodule init third_party/pybind
-          git submodule update third_party/pybind
-
       - name: Build wheels
         run: python -m cibuildwheel loadgen/ --output-dir wheels
         env:
           CIBW_ENVIRONMENT: "CFLAGS='-std=c++14'"
-          CIBW_BUILD: 'cp3{6,7,8,9,10}-*'
+          CIBW_BUILD: 'cp3{6,7,8,9,10,11}-*'
```

Comment:
Do we still need python36 (EOL at Dec 2021)?

Reply:
MLPerf power needs 3.7+, so may be we should remove 3.6 support.

Reply:
sure. do you want me to remove it? or you can force-push it easily.

Reply:
@pgmpablo157321 is the right person to answer that :)

Reply:
Hi @ljk3210. Yes, could you remove python 3.6?

Reply:
Done.